addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11261,7 +11261,11 @@ msgctxt "#19685"
 msgid "Confirm shutdown"
 msgstr ""
 
-#empty string with id 19686
+#: Label for buttons to open channel guide dialog
+#: addons/skin.estuary/1080i/DialogPVRInfo.xml
```

Comment:
Right, but what speaks against doing it right in the first place for new stuff we're adding? ;-)